mattumland/Salon
# Template merge

## Is this a fix or a feature?
Feature
## What is the change?
This adds the Wall and Artwork components and connects them to the data being fetched in App. It renders then with some initial salon styling but it isn't finalized. I will create a new card for future improvement to the salon styling. 
## What does it fix?

## Where should the reviewer start?
Review the new files (Wall.js/scss Artwork.js/scss). Neither are very long. 
## How should this be tested?
Run the dev server, watch how it loads. Try reloading a few times to see how it different collections of art appear.
